### PR TITLE
docs(accessibility): add decsriptions to components and behaviors

### PR DESCRIPTION
**Pull request by jurokapsiar**
Add descriptions to components and behaviors
In components, put @accessibility description first, before behaviors
In behaviors, explicitly split description and specification of behaviors

### CODE REVIEW

File: src/lib/accessibility/Behaviors/Toolbar/toolbarBehavior.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Accessibility, FocusZoneMode } from '../../types'
 
 /**
  * @specification
```
Comment (hughreeling) [inline src/lib/accessibility/Behaviors/Toolbar/toolbarBehavior.ts]: @description?  Or is there a difference?

Comment (jurokapsiar) [reply src/lib/accessibility/Behaviors/Toolbar/toolbarBehavior.ts]: good catch. difference is that lines from specification are unit tested.

File: src/lib/accessibility/Behaviors/Radio/radioGroupBehavior.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { Accessibility } from '../../types'
 import * as keyboardKey from 'keyboard-key'
 
 /**
+ * @description
+ * Implements ARIA Rafio Group design pattern.
```
Comment (sophieH29) [inline src/lib/accessibility/Behaviors/Radio/radioGroupBehavior.ts]: Rafio -> Radio

File: docs/src/components/DocsBehaviorRoot.tsx [hunk 3]
```diff
@@ -69,8 +69,11 @@ class DocsBehaviorRoot extends React.Component<any, any> {
                   </div>
                 </Grid.Column>
                 <div style={{ padding: '1em' }}>
-                  <span> Description: </span>
-                  <br />
+                  {variation.description && (
```
Comment (miroslavstastny) [inline docs/src/components/DocsBehaviorRoot.tsx:72]: merge the two conditions to one (+ the same for `variation.specification`)

File: docs/src/components/DocsBehaviorRoot.tsx [hunk 4]
```diff
@@ -69,8 +69,11 @@ class DocsBehaviorRoot extends React.Component<any, any> {
...
+                    <>
+                      <strong>Description:</strong> <br />
+                    </>
+                  )}
                   {variation.description &&
                     variation.description.split('\n').map((splittedText, keyValue) => {
                       return (
```
Comment (miroslavstastny) [inline docs/src/components/DocsBehaviorRoot.tsx]: `=> { return () }` -> ` => ()`

**Review by miroslavstastny — APPROVED** [review]
add changelog

**Review by sophieH29 — APPROVED** [review]

**Pull request merged**